content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,139 @@
+{{< img src="getting_started/integrations/integration-tile-status.png" alt="Datadog AWS integration tile with 'Account credentials are valid' message">}}
+
+5. Finally, click `Install integration`. For the purposes of this demonstration, all options have been selected (only `Collect custom metrics` is unticked by default):
+{{< img src="getting_started/integrations/integration-tile-complete.png" alt="Datadog AWS integration tile">}}
+
+It should be noted that there are multiple options within the integration tile to specify the data to be collected. This can be limited based on AWS service, namespace, or tags.
+With [EC2 automuting][10] enabled, Datadog uses host statuses reported by the CloudWatch API to distinguish between expected and unexpected instance terminations, preventing unwanted Monitor notifications.
+
+### Validation
+
+The automatically-provisioned [AWS overview dashboard][11] provides an immediate window into the metrics sent by your AWS services and infrastructure:
+{{< img src="getting_started/integrations/aws-dashboard.png" alt="AWS overview dashboard">}}
+
+
+## Sending Logs
+
+As a result of setting up the Datadog Forwarder function with the CloudFormation template, log collection can be configured from directly within the integration tile in Datadog. Click into the `Collect Logs` tab in the Integration tile and enter the ARN of the Forwarder lambda. With the ARN entered, click the check mark to establish the connection: 
+{{< img src="getting_started/integrations/integration-tile-collect-logs-1.png" alt="Datadog AWS integration tile 'Collect Logs' tab">}}
+
+Once completed, you'll be able to select from any of the log sources mentioned in [this guide][12] for sending AWS service logs. For this guide, we'll select only S3 access logs, and click `Save`:
+{{< img src="getting_started/integrations/integration-tile-collect-logs-2.png" alt="Datadog AWS integration tile 'Collect Logs' tab with 'S3' option checked">}}
+
+
+### Validation
+
+Find your logs in the [Logs Explorer][13] using either the `source` or `service` facets from the facet panel:
+{{< img src="getting_started/integrations/logs-explorer.png" alt="Datadog Logs Explorer">}}
+
+## Deeper visibility with the Datadog agent
+
+By default the Datadog AWS integration crawls the CloudWatch API for AWS-provided metrics, but you can gain even deeper visibility into your EC2 instances and ECS clusters with the [Datadog agent][14]. The agent is a lightweight daemon which reports metrics and events, and can also be configured for logs and traces. The [Agent Installation][15] section of the Datadog application provides instructions for installing the agent on a wide variety of operating systems, with many options--such as Amazon Linux 2--offering one-step installation commands. This command can then be run from the instance terminal to install the agent:
```

Comment:
```suggestion
By default the Datadog AWS integration crawls the CloudWatch API for AWS-provided metrics, but you can gain even deeper visibility into your EC2 instances and ECS clusters with the [Datadog Agent][14]. The agent is a lightweight daemon which reports metrics and events, and can also be configured for logs and traces. The [Agent Installation][15] section of the Datadog application provides instructions for installing the Agent on a wide variety of operating systems, with many options--such as Amazon Linux 2--offering one-step installation commands. This command can then be run from the instance terminal to install the Agent:
```

Reply:
`The Agent is a lightweight daemon that reports metrics and events` (not which)